scripts/dev/build_release.py

```diff
@@ -387,10 +400,15 @@ def main():
         artifacts = build_sdist()
         upload_to_pypi = True
 
-    if args.upload is not None:
+    if args.upload:
+        from qutebrowser import __version__
         utils.print_title("Press enter to release...")
         input()
-        github_upload(artifacts, args.upload[0])
+
+        # create version tag
```

Comment:
That comment confused me, as no tag is actually created (as in `git tag`) - I think it'd be best to remove it :wink: